server/controllers/lists/remove_selections.js

```diff
@@ -0,0 +1,36 @@
+const _ = require('builders/utils')
+const lists_ = require('controllers/lists/lib/lists')
+const { filterFoundSelectionsUris } = require('controllers/lists/lib/helpers')
+const error_ = require('lib/error/error')
+const selections_ = require('controllers/lists/lib/selections')
+const { addWarning } = require('lib/responses')
+
+const sanitization = {
+  id: {},
+  uris: {}
+}
+
+const controller = async ({ id, uris, reqUserId }, req, res) => {
+  const list = await lists_.getListWithSelections(id, reqUserId)
+  if (!list) throw error_.notFound({ id })
+
+  lists_.validateOwnership(reqUserId, list)
+
+  const selectionsToDelete = []
+  const notFoundUris = []
+  filterFoundSelectionsUris(list.selections, uris, selectionsToDelete, notFoundUris)
+  if (!_.isNonEmptyArray(selectionsToDelete)) {
```

Comment:
```suggestion
  if (selectionsToDelete.length === 0) {
```

Reply:
fixup! add Selection as sub-List element

to not block the PR